### ISSUE TITLE
Sort error displaying landing pages

### ISSUE THREAD

**Issue by ryscher** [CDL-Dryad/dryad-product-roadmap#2389]
We are seeing errors when people try to access landing pages:
```
 Mysql2::Error: Out of sort memory, consider increasing server sort buffer size
 app/views/stash_engine/landing/_files.html.erb:28
```

**Comment by ryscher** [comment]
This appears to be an oddity of MySQL when working with tables that include a JSON field.

The query that is generated to display data files on the landing page looks like:
```
SELECT `stash_engine_generic_files`.* FROM `stash_engine_generic_files` INNER JOIN (SELECT max(id) last_id, upload_file_name FROM `stash_engine_generic_files` WHERE `stash_engine_generic_files`.`type` = 'StashEngine::DataFile' AND `stash_engine_generic_files`.`resource_id` = 150 AND (file_state <> 'deleted' AND (url IS NULL OR (url IS NOT NULL AND status_code = 200))) GROUP BY `stash_engine_generic_files`.`upload_file_name`) sub on id = sub.last_id WHERE `stash_engine_generic_files`.`type` = 'StashEngine::DataFile' ORDER BY `stash_engine_generic_files`.`upload_file_name` ASC;
```

This query fails in the production database. However, it works fine if you change the SELECT portion to include specific fields (other than the `container_contents` JSON field).

Possible solutions:
- ask IAS to increase the sort buffer size
- rework the code to only query for the fields we need

**Comment by sfisher** [comment]
This is triaged for for now by removing all the contents of the container_contents field.

Audrey has the good suggestion of moving this field into its own table so it doesn't get regularly included in queries unless something specifically wants it and JOINs to it or loads it separately.

I'll work on refactoring to move this out of the table.

**Comment by ryscher** [comment]
Closing this, since the immediate problem is resolved, and the underlying fix is in progress with #2390.
